Gemfile

```diff
@@ -54,7 +62,10 @@ group :development do
 
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
+  gem 'pry', '~> 0.10.4'
+  gem 'better_errors'
 
+  gem 'rb-readline', platforms: :ruby, install_if: os_is(/darwin/)
```

Comment:
Are we even using rb-readline here? Also it seems to be a gem that any OS can use? Why limit it to OSX?

https://github.com/ConnorAtherton/rb-readline